src/imgui/ImGuiCharacter.cc

```diff
@@ -412,6 +413,24 @@ void ImGuiCharacter::paint(MSXMotherBoard* motherBoard)
 				printAddressRange8("Color", colTable.getAddress((mode == SCR1) ? (pattern / 8) : (8 * pattern)));
 			}
 		};
+		auto formatBinaryData = [&](Debuggable& debuggable, uint16_t address) {
+			return formatToString(debuggable, address, address + 7, "", 1, "", "%02X", manager.getInterpreter());
```

Comment:
Very much a detail: instead of passing empty strings as `""` is a tiny bit better to pass `{}`.

The former calls the constructor that takes a `const char*`, that constructor calls `strlen()` and figures out it's an empty string. The latter calls the default constructor (the constructor without parameters) and that one directly constructs an empty string.

Compilers like gcc and clang optimize the call to strlen() away, so both variants generate the same code. But msvc is less smart and there using `{}` does generate better code.